index.html

```diff
@@ -11,9 +11,23 @@
 
 </head>
 
+<header>
+    <nav>
```

Comment:
Did you find a different lambda image to use?  Also, you can use the assets in the assets folder in your project instead of going out to the internet to get image files.